src/CMakeLists.txt

```diff
@@ -91,7 +92,7 @@ target_include_directories(d3d12translationlayer
 source_group(Inlines FILES ${INL})
 source_group("Header Files\\External" FILES ${EXTERNAL_INC})
 
-target_link_libraries(d3d12translationlayer d3d12 dxcore dxgi atls)
+target_link_libraries(d3d12translationlayer d3d12 dxcore dxgi atls dxbcparser dxbcsigner)

```

Comment:
I don't think we want to link to the signer at this level. 11on12 and CLon12 don't need a DXBC signer at all, since they use DXIL.